api/controller.py

```diff
@@ -628,24 +625,6 @@ def authentication_for_opds_document(self):
             value = json.dumps(value)
         return value
 
-    @property
```

Comment:
This is identical to the `key_pair` implementation for the `authentication_document` [here](https://github.com/ThePalaceProject/circulation/blob/feature/remove-sitewide-key/api/authenticator.py#L1200).
Does it server the same purpose? Should it also be removed?

Reply:
It would be nice to be able to remove that one as well, but the library specific keypair in the authentication_document is used by the library registry to verify the identity of the library, so it can't just be dropped like this one. 

This served the same purpose, but for the metadata wrangler integration. But since that integration is gone, this one can be removed.